x-pack/plugins/cloud_security_posture/public/pages/rules/rules_container.tsx

```diff
@@ -77,12 +79,31 @@ export const RulesContainer = () => {
     [data, error, status, rulesQuery]
   );
 
+  // We need to make this call again without the filters. this way the section list is always full
```

Comment:
the cleanest would be to either have a route on the server that returns the sections or do all the filtering/sorting on the client, but I guess we don't want to invest much into the rules page. I hope it won't bite us in the future :)